.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: python
+python:
+    - "2.7"
+install:
+- "pip install pytest"
+- "pip install python-dateutil"

```

Comment:
if I remember correctly, `pip install .` should use dependencies from `setup.py`